submissions/Roophee/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,428 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="img/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="img/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <link rel="mask-icon" href="img/safari-pinned-tab.svg" color="#5bbad5">
+    <meta name="msapplication-TileColor" content="#2d89ef">
+    <meta name="theme-color" content="#ffffff">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Cinema</title>
+</head>
+<body>
+<div class="container">
+    <header class="header">
+        <div class="header__container">
+            <nav class="header__nav">
+                    <ul class="header__nav-list">
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">Главная</a>
+                        </li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">О&#160;нас</a></li>
+                        <li  class="header__nav-item">
+                            <a href="#" class="header__nav-item">Сеансы&#160;и&#160;Билеты</a></li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">Контакты</a></li>
+                        <li class="header__nav-item">
+                            <a href="#" class="header__nav-item">Форум</a></li>
```

Comment:
Increase font-size on mobile.

Reply:
Done. Please, tell me how it looks.
Thanks.

Reply:
Looks great for me.